kythe/cxx/indexer/cxx/IndexerASTHooks.h

```diff
@@ -143,6 +143,8 @@ struct IndexerOptions {
       CreateWorklist = [](IndexerASTVisitor* indexer) {
         return IndexerWorklist::CreateDefaultWorklist(indexer);
       };
+  /// \brief Notified each time a semantic signature is hashed.
+  std::unique_ptr<HashRecorder> HashRecorder;
```

Comment:
This makes this struct no longer copyable (which might be fine), but this also seems more natural as an unowned pointer.